build/gitlab/stages/3-cluster-connect/cluster-connect

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+time="$(TZ=IST date)"
+current_time=$time
+echo $current_time
+path=$(pwd)
+
+printf "***Conneting to the Cluster*****\n\n"
+
+##Conneting to the Cluster
+go="sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'ls | grep go'"
```

Comment:
This is not the correct way to check the go installed or not
use these command:
1 - go version
2 - which go
3 - whereis go

Reply:
Not checking that the go is installed 
checking if go directory is there or not

Reply:
Do we need that directrory?

Reply:
Yes as go path is exported in /go
and further git clone will be done inside go the path

Reply:
Generalized it more by creating `GOPATH` if  go path will not be there, then it will create one

Reply:
These are all risks/dependence on landing machine..as we have discussed before. Lets create an issue to m make it machine agnostic.